pkg/server/server.go

```diff
@@ -44,6 +44,10 @@ func NewServer(host string, port int, insecureSkipTLS bool, backends ...string)
 	r.Use(ginzap.RecoveryWithZap(zapLogger, true))
 	r.GET("/", handleHealthz)
 	r.POST("/", sprayProxy.HandleProxy)
+
+	r.POST("/register", sprayProxy.RegisterBackends)
+	r.GET("/unregister", sprayProxy.UnregisterBackends)
```

Comment:
Using GET method to delete or update data don't make sense.  How do you think we use  DELETE or PUT?

Reply:
It will get the asked url from url list and delete that. Entire definition is here https://github.com/prietyc123/sprayproxy/pull/1/files#diff-b41d02f71c670ff801d99892d32ef6dd36e43fcdf4c12c044cd34a073cc18200R57